modules/nf-core/simpleaf/quant/main.nf

```diff
@@ -0,0 +1,68 @@
+process SIMPLEAF_QUANT {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::simpleaf=0.13.0"
```

Comment:
In INDEX we use 0.14.0, and here we use 0.13.0. Maybe use only 0.13.0 for all of them, or 0.14.0

Reply:
I agree with @Joaodemeirelles, better to be consistent with the version

Reply:
Well spotted, will do